docs/learn-account-restore.md

```diff
@@ -25,96 +25,72 @@ your account. This part is _very important_.
-main page of the plugin. There listed will be your restored account!
+main page of the plugin. There listed will be your restored account.
 
-## Polkadot.js
+## Polkadot{.js}
 
-If you're using the main Polkadot.js UI, restoring an account will feel similar to restoring an
-account on Polkadot.js browser plugin. The first step is to head over to the chosen account and
-click on the three dots beside it. This will open up a tab with lots of options. For backing up the
-account we'll restore later, press on "Create a backup file for this account".
+If you're using the main Polkadot{.js} UI, restoring an account will feel similar to restoring an
+account on Polkadot{.js} browser plugin. The first step is to click on the "Restore JSON" button.
 
 ![click-account-restore-polkadot](assets/accounts/polkadot_click_restore.png)
 
-From there, a pop up screen will appear, prompting for the password of that account. By entering
-your password and clicking on "Download", your browser will download the `.json` file associated
-with the Polkadot account. This `.json` file holds all relevant data about the account to be used in
-account restoration.
-
-![enter-account-password-polkadot](assets/accounts/polkadot_enter_password.png)
-
 Now that's completed, it's time to restore the account. On the Polkadot accounts page, click on the
 "Restore JSON" button, which will let you upload your `.json` file that you downloaded and enter
-your password for that account.
+your password for that account. This `.json` file holds all relevant data about the account to be
+used in account restoration.
 
 ![restore-account-polkadot](assets/accounts/polkadot_restore_account.png)
 
 Once you've pressed the "Restore" button, you should see a green notification letting you know that
-your account has been restored. That's it!
+your account has been restored.
 
 ![restore-complete-polkadot](assets/accounts/polkadot_restore_complete.png)
 
 ### Using an existing Mnemonic Seed to restore an Account
 
 If you've already created an account but are looking to have it restored, you can do that by using
-the Polkadot.js UI and the existing mnemonic seed.
+the Polkadot{.js} UI and the existing mnemonic seed.
 
-To start, you'll need to open up the [Polkadot.js UI](https://polkadot.js.org/apps) and head over to
-the accounts tab. Click on the "Add Account" button, and fill out with the prompted details: a name
-for the account, a password, and be sure to replace the generated mnemonic seed with your _existing
-seed_. Note here that when you replace the generated mnemonic seed with your existing one, you
-aren't actually creating a new account, rather adding that account onto the Polkadot.js UI. It is
-also important to note that the name and the password of this added account can be set to whatever
-you'd like, it does not need to be that specific accounts name and password when created.
+To start, you'll need to open up the [Polkadot{.js} UI](https://polkadot.js.org/apps) and head over
```

Comment:
```suggestion
To start, you'll need to open up the [Polkadot{.js} Apps](https://polkadot.js.org/apps) and head over
```

We need to converge on a consistent spelling for Polkadot-JS across the wiki. I think either Polkadot-JS or Polkadot{.js} works, although the second one might be more surprising for a reader to encounter in prose documentation. Also Apps vs UI, my vote is to use Apps across the wiki since that's the name of the repository while the [UI repo](https://github.com/polkadot-js/ui) contains browser libraries and React components.

Reply:
^^ We'll take care of this in another PR